uber/site_sections/preregistration.py

```diff
@@ -260,7 +260,7 @@ def dealer_confirmation(self, session, id):
     @log_pageview
     def group_members(self, session, id, message=''):
         group = session.group(id)
-        charge = Charge([group, group.leader]) if group.leader else Charge(group)
+        charge = Charge([group] + listify(group.attendees))
```

Comment:
Is this change required? If we use `group.attendees` below, then we shouldn't need to add each attendee as a target for the `Charge`, correct?